resources/lang/en/patrols/desert/hunting/any.json

```
@@ -2459,5 +2459,350 @@
+        "decline_text": "Just before the patrol could begin their work, p_l suddenly decides against this endeavor. There can be better prey to find than one or two ground squirrels, {PRONOUN/m_c/subject} {VERB/m_c/argue/argues}.",
+        "weight": 20,
+        "chance_of_success": 50,
+        "tags": [],
+        "success_outcomes": [
+            {
+                "text": "The patrol darts forward towards the stream at p_l's command, and it was not long until the spotted prey was surrounded. p_l watches with pride as the others begin piling their prey up for the return trip to camp.",
+                "exp": 20,
+                "weight": 20,
+                "prey": ["medium"]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "p_l and the patrol rush forward at {PRONOUN/m_c/poss} command, but the ground squirrels were ever faster. The prey fell back into their den holes by the time the cats made it to the stream.",
+                "exp": 0,
+                "weight": 20,
+                "prey": ["very_small"]
+            }
+        ]
+    },
+    {
+        "patrol_id": "dst_hunt_kangratsolo",
+        "biome": [
+            "desert"
+        ],
+        "season": [
+            "any"
+        ],
+        "types": [
+            "hunting"
+        ],
+        "min_cats": 1,
+        "max_cats": 1,
+        "patrol_art": null,
+        "intro_text": "p_l steps lightly towards a small dune, spotting a set of bouncing tracks in the sand. ",
+        "decline_text": "With a huff, p_l decides that {PRONOUN/m_c/subject} could be tracking something better, and leaves the tracks alone.",
+        "weight": 20,
+        "chance_of_success": 50,
+        "tags": [],
+        "success_outcomes": [
+            {
+                "text": "p_l lowers {PRONOUN/m_c/self} to the ground and goes the other way behind the dune, surprising and trapping a kangaroo rat! ",
+                "exp": 20,
+                "weight": 20,
+                "prey": ["small"]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "p_l attempts to follow the tracks, but {PRONOUN/m_c/poss} footsteps cause the kangaroo rat {PRONOUN/m_c/subject} {VERB/m_c/were/was} hunting to dart away quickly!",
+                "exp": 0,
+                "weight": 20,
+                "prey": ["very_small"]
+            }
+        ]
+    },
+    {
+        "patrol_id": "dst_hunt_kangratwar",
+        "biome": [
+            "desert"
+        ],
+        "season": [
+            "any"
+        ],
+        "types": [
+            "hunting"
+        ],
+        "min_cats": 2,
+        "max_cats": 6,
+        "patrol_art": null,
+        "intro_text": "p_l and {PRONOUN/m_c/poss} follow the tracks left behind by a colony of bouncing kangaroo rats. r_c seems especially eager to hunt them down with a grumble in {PRONOUN/m_c/poss} belly.",
```

```suggestion
        "intro_text": "The desert cats follow tracks left behind by a feeding aggregation of bouncing kangaroo rats. r_c seems especially eager to hunt them down with a grumble in {PRONOUN/r_c/poss} belly.",
```

(kangaroo rats aren't actually social animals)